docs/guides/capabilities/ai/meeting-transcription.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+---
+
+# Meeting Transcription
+
+Dyte's meeting transcription allows you to transcribe your Dyte meetings in real-time, making it easy to capture important discussions and refer back to them later.
+This guide will walk you through how to use this feature effectively.
+
+## Control transcriptions for participants using presets
+
+You can control whether or not a participant's audio will be transcribe with the help of the `transcription_enabled` flag in the participant's preset.
+All participants with the `transcription_enabled` turned on in their preset will be able to generate transcripts in real-time in a Dyte meeting.
+
+You can create a new preset on our [Developer Portal](https://dev.dyte.io/presets), or using our [REST API](https://docs.dyte.io/api#/operations/post-presets)
+
+## Consuming transcripts
+
+There are 2 ways in which these transcripts can be consumed.
```

Comment:
For the bullet points, the first words can be used to indicate the way in which the transcript can be consumed to make it easy to glance through. Something like: 

> 1. Client coreSDK : The transcripts can be consumed on the client-side using the Dyte SDK that's suitable for your platform. These transcripts are generated on the server in real-time. 

>2. Webhooks: The meeting transcript can be consumed via a [webhook after the meeting ends](#consume-transcript-via-a-post-meeting-webhook).